Anime-App/Base/Extensions/String+capitalizingFirstLetter.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  String+.swift
```

Comment:
I would actually call this file String+Manipulation, the second part should just describe what the extension is doing at a high level not go into too much specifics